src/logic/operate.js

```diff
@@ -0,0 +1,32 @@
+import Big from 'big.js';
+
+const operate = (numOne, numTwo, operation) => {
+  const first = Big(numOne);
+  const second = Big(numTwo);
+
+  let answer;
+
+  switch (operation) {
+    case '+':
+      answer = first.plus(second);
+      break;
+
+    case '-':
+      answer = first - second;
+      break;
+
+    case 'x':
+      answer = first * second;
+      break;
+
+    case '/':
+      answer = first / second;
+      break;
```

Comment:
- [ ] Please use `Big.js` functions `minus`, `mul` and `div` to make those operations, since you are using the library.